### PR TITLE
GROW-574 Fixes ThanksPage V2 experiment when user is not a guest

### PR DESCRIPTION
**Pull request by eddieferrer**

### CODE REVIEW

File: src/pages/Thanks/ThanksPage.vue [hunk 1]
```diff
@@ -188,7 +188,16 @@ export default {
 			// But it will not throw a server error.
 			this.receipt = data?.shop?.receipt;
 			this.isGuest = this.receipt && !data?.my?.userAccount;
-			this.thanksPageVersion = this.isGuest ? 'b' : 'a';
```
Comment (eddieferrer) [inline src/pages/Thanks/ThanksPage.vue:191]: This line seemed to be overwriting the change in the mounted block for some reason, in my testing result sometimes finished after the mounting code... kind of puzzling.

**Review by mcstover — APPROVED** [review]

File: src/pages/Thanks/ThanksPage.vue (hunk 1 above)
Comment (emuvente) [reply src/pages/Thanks/ThanksPage.vue:191]: result is also called everytime the watchQuery call has a new result, so it could be that

**Review by emuvente — APPROVED** [review]
👍

File: src/pages/Thanks/ThanksPage.vue (hunk 1 above)
Comment (eddieferrer) [reply src/pages/Thanks/ThanksPage.vue:191]: @emuvente good call, I didn't think about that!

**Pull request merged**